streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImplTest.java

```diff
@@ -1044,7 +1043,7 @@ public synchronized long position(final TopicPartition partition) {
     }
 
     @Test
-    public void shouldUseRequestTimeoutPlusTaskTimeoutInPollDuringRestoreAndFailIfNoDataReturned() {
+    public void shouldUsePollMsPlusRequestTimeoutInPollDuringRestoreAndTimeoutWhenNoProgressDuringRestore() {
```

Comment:
After the refactoring who we handle `task.timeout.ms` during global restore, this test and the next test did cover the exact some thing. So I removed one test to avoid redundancy.